test/unit/ActionBar.spec.js

```diff
@@ -0,0 +1,66 @@
+import { mount, createLocalVue } from '@vue/test-utils';
+import BootstrapVue from 'bootstrap-vue';
+import ActionBar from '../../src/components/ActionBar.vue';
+
+const localVue = createLocalVue();
+localVue.use(BootstrapVue);
+
+test('Component is mounted properly', () => {
+	const wrapper = mount(ActionBar, {
+		localVue,
+	});
+	expect(wrapper).toMatchSnapshot();
+});
+
+test('if the dismiss icon is shown when the prop isDismissible is setted to true', () => {
+	const wrapper = mount(ActionBar, {
+		localVue,
+		propsData: {
+			isDismissible: true,
+		},
+	});
+
+	expect(wrapper.findAll('.icon-container').length).toBe(1);
+});
+
+test("if the 'fixed' class is used when the floatingMode is setted to true", () => {
+	const wrapper = mount(ActionBar, {
+		localVue,
+		propsData: {
+			floatingMode: true,
+		},
+	});
+
+	expect(wrapper.findAll('.fixed').length).toBe(1);
+});
+
+test('if the event is emited correctly when the dismissible icon is clicked', () => {
+	const wrapper = mount(ActionBar, {
+		localVue,
+		propsData: {
+			isDismissible: true,
+		},
+	});
+
+	wrapper.find('.icon-container').trigger('click');
+
+	expect(wrapper.emitted().close).toBeTruthy();
```

Comment:
Qual a diferença da linha 47 pra linha 48? Em termos do que está sendo testado.

Reply:
The line 47 asserts that the event has been emitted properly and the line 48 asserts that the value it emitted is equal to'true'